src/main/java/org/yaml/snakeyaml/LoaderOptions.java

```
@@ -22,6 +22,7 @@
     private int maxAliasesForCollections = 50; //to prevent YAML at https://en.wikipedia.org/wiki/Billion_laughs_attack
     private boolean allowRecursiveKeys = false;
     private boolean processComments = false;
+    private boolean enumCaseSensitive = false;
```

I think we should use `true` here because that is how it was working before.

With default `false` we are changing expectations.

---

Thats right ;) Fixed

---

but don't you need also the test ?
Now default Yaml will be enumCaseSensitive = true and it should fail 1st test (you are not setting it to false).
Or did I miss something?

---

Yes indeed. I forgot about it. Fixed!